packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -12,6 +12,7 @@ const selectors = {
 
 	// Buttons and links within Sidebar
 	linkWithText: ( text: string ) => `a:has-text("${ text }")`,
+	planName: '.sidebar__inline-text',
```

Comment:
This works for now, but I'm a little hesitant about relying on only a stylistic class name. Could we perhaps position this within the "Upgrades" row?

Like `.sidebar__menu:has-text("Upgrades") .sidebar__inline-text`

Reply:
That's fair. I'll use the same selector I do in other part of `SidebarComponent` to narrow it down a bit more.